www/explore/teams/index.spt

```diff
@@ -27,18 +18,18 @@ subhead = _("Teams")
 <h2>{{ _("Teams (N={0})", nteams) }}</h2>
 <table class="table">
     <tr>
-        <th>{{ _("Team Name") }}</th>
-        <th class="figure">{{ _("Receiving ($)") }}</th>
-        <th class="figure">{{ _("Members") }}</th>
-        <th class="figure">{{ _("Receiving{0} Per Member ($)", "<br>"|safe) }}</th>
+        <th>{{ _("Name") }}</th>
+        <th>{{ _("Owner") }}</th>
+        {% if user.ADMIN %}<th>{{ _("Status") }}</th>{% endif %}
     </tr>
     {% for team in teams %}
+    {% if team.is_approved or user.ADMIN %}
     <tr>
-        <td class=""><a href="/{{ team.name }}/">{{ team.name }}</a></td>
-        <td class="figure">{{ format_decimal(team.receiving, '#,##0.00') }}</td>
-        <td class="figure">{{ team.nmembers }}</td>
-        <td class="figure">{{ format_decimal(team.dollars_per_member, '#,##0.00') }}</td>
+        <td><a href="/{{ team.slug }}/">{{ team.name }}</a></td>
+        <td><a href="/{{ team.owner }}/">{{ team.owner }}</a></td>
+        <td>{{ team.status }}</td>
```

Comment:
Isn't the status column useless if all the teams shown are `approved`? Shouldn't this be shown only to admins?